README.md

```diff
@@ -4,8 +4,7 @@
 
 # Terraform Modules
 
-This repo contains [Terraform](https://www.terraform.io) modules for running a Private Kubernetes cluster on [Google Cloud Platform (GCP)](https://cloud.google.com/)
-using [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/).
+This repo contains [Terraform](https://www.terraform.io) modules for running a Private Kubernetes cluster.
```

Comment:
Why does this have to be limited to a private kubernetes cluster?

Reply:
As we can add any module, there is no reason for that. Fixing it now.